services/movie.py

```diff
@@ -0,0 +1,45 @@
+from django.db.models import QuerySet
+
+from db.models import Movie
+
+
+def get_movies(
+        genres_ids: list = None,
+        actors_ids: list = None
+) -> QuerySet[Movie]:
+    query = Movie.objects.all()
+
+    if genres_ids and actors_ids:
+        query = query.filter(
+            genres__id__in=genres_ids,
+            actors__id__in=actors_ids
+        )
+
+    if genres_ids and not actors_ids:
+        query = query.filter(genres__id__in=genres_ids)
+
+    if actors_ids and not genres_ids:
+        query = query.filter(actors__id__in=actors_ids)
```

Comment:
Leave only two if conditions and append new `filter` methods to queryset instead of looking for all combinations. In case there is greater number of arguments, there would be a progressing increase in the number of if conditions